lib/AST/StmtPrinter.cpp

```diff
@@ -1735,6 +1735,46 @@ void StmtPrinter::VisitAtomicExpr(AtomicExpr *Node) {
   OS << ")";
 }
 
+// Checked C extension
+
+void StmtPrinter::VisitCountBoundsExpr(CountBoundsExpr *Node) {
+  switch (Node->getKind()) {
+    case CountBoundsExpr::Kind::Byte_Count:
+      OS << "byte_count(";
+    case CountBoundsExpr::Kind::Count:
+      OS << "count(";
+    default:
+      llvm_unreachable("count bounds kind not covered by switch");
+      break;
+  }
+  PrintExpr(Node->getCountExpr());
+  OS << ")";
+}
+
+void StmtPrinter::VisitNullaryBoundsExpr(NullaryBoundsExpr *Node) {
+  switch (Node->getKind()) {
+  case NullaryBoundsExpr::Invalid: {
+    OS << "bounds(invalid)";
```

Comment:
This is an AST printer, so we should just print out the AST, not give an error, even if the the kind is Invalid.